machinist/machine/commands.go

```diff
@@ -43,6 +43,7 @@ func NewEnrollCommand(conf *config.Node) *cobra.Command {
 	}
 	// General Flags.
 	c.PersistentFlags().BoolVar(&conf.RequireRoot, "require-root", true, "should the enroll command require root for execution")
+	c.PersistentFlags().BoolVar(&conf.ModifyMachineConfig, "modify-machine-config", false, "perform additional setup (install libnss_autouser and config, PAM scripts, etc.)")
```

Comment:
can you default the flag to true? on puppet machines, it's easy to set this flag to a different value. But at least it won't break the config/setup of non-puppet machines?

Reply:
done